rtf

```diff
@@ -368,6 +417,170 @@ def list_scenarii():
     if file != "__init__" and file != "lib":
       print(file)
 
+def make_rule_testfile(filename, rule, directives):
```

Comment:
For string interpolation do you think using the "locals" variable would help ? like here:

http://stackoverflow.com/questions/4450592/string-interpolation-in-python#answer-4450610

So you don't have to redefine every variable in your interpolation dict